RELEASE_NOTES.md

```diff
@@ -8,6 +8,21 @@ redirect_from:
 ---
 # Version history
 
+## Version 0.8.1
+
+April 11th 2025: 🧑‍🤝‍🧑 The 18th draft changes to community governance model.
```

Comment:
I see that we have uncovered another oversight in the releasing documentation!

The date should be `DATE-OF-RELEASE`, so that it can be [set by the automation](https://github.com/standard-for-public-code/standard-for-public-code/blob/develop/script/update-release-notes-date.sh#L46).

Reply:
Aha! Perhaps this is correct?

```suggestion
DATE-OF-RELEASE: 🧑‍🤝‍🧑 The 18th draft changes to community governance model.
```

Reply:
Typo fixed in https://github.com/standard-for-public-code/standard-for-public-code/pull/1096/commits/7c12ccc36c7cc2bf498f5ca875874e5626c3cda0